neuralprophet/conformal.py

```diff
@@ -167,29 +191,44 @@ def plot(self, plotting_backend: str):
                 specifies the plotting backend for the nonconformity scores plot, if any
 
                 Options
-                    * ``matplotlib``: Use matplotlib backend for plotting
-                    * ``plotly``: Use the plotly backend for plotting
+                * ``plotly-resampler``: Use the plotly backend for plotting in resample mode. This mode uses the
+                    plotly-resampler package to accelerate visualizing large data by resampling it. For some
+                    environments (colab, pycharm interpreter)plotly-resampler might not properly vizualise the figures.
+                    In this case, consider switching to 'plotly-auto'.
+                * ``plotly``: Use the plotly backend for plotting
+                * ``matplotlib``: use matplotlib for plotting
+                * (default) None: Plotting backend ist set automatically. Use plotly with resampling for jupyterlab
+                    notebooks and vscode notebooks. Automatically switch to plotly without resampling for all other
+                    environments.
 
         """
         method = self.method.upper() if "cqr" in self.method.lower() else self.method.title()
-        if plotting_backend == "plotly":
+        # Check whether a local or global plotting backend is set.
+        plotting_backend = auto_set_plotting_backend(plotting_backend)
+        if hasattr(self, "plotting_backend"):
+            plotting_backend = auto_set_plotting_backend(self.plotting_backend)
+
+        log_warning_deprecation_plotly(plotting_backend)
+        if "plotly" in plotting_backend.lower():
```

Comment:
Also `startswith()` might be the more accurate choice over `in`
```suggestion
        if plotting_backend.lower().startswith("plotly"):
```